### PR TITLE
[BD] Drop python 2.7 support

### PR DESCRIPTION
**Pull request by morenol**
## Description

Drop python 2.7 support. Fix tox configuration.

Note: Check if the reporting scripts are being used with python3.

## Reviewers

- [ ] @andrey-canon
- [x] Ready for edX review
- [ ] @jmbowman

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4293](https://openedx.atlassian.net/browse/OSPR-4293) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by morenol** [comment]
@jmbowman In the last PR, I made an error, the tox tests are not running the actual commands. This also fixes that. I made that change also in https://github.com/edx/edx-enterprise-data/pull/183 in case you only want to approve that change and wait a little more for the python 2.7 deprecation

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
+  - 3.5
 env:
   - TOXENV=django111-data
   - TOXENV=django111-reporting
 matrix:
   include:
-  - python: 3.6
-    env: TOXENV=quality
-  - python: 2.7
     env: TOXENV=quality
```
Comment (jmbowman) [inline .travis.yml]: It looks like this can now just be added to the `env:` list above, getting rid of `matrix` completely.

**Review by jmbowman — APPROVED** [review]
Looks good, just holding off on merging until Insights has proven itself stable on 3.5 in prod for a couple of days.

**Comment by jmbowman** [comment]
I checked, and the reporting scripts have already been switched to use Python 3.5.

**Pull request merged**